### DISCUSSION TITLE
希望作者大大能出一个获取规则的app 不然自己添加不了

### DISCUSSION THREAD

**Discussion by azy00728** [xiaohucode/yidaRule#50] [Ideas]
反馈目前66个有几个不能用了 第一个维特 第二个哔哩哔哩 第三个 禁漫天堂 首页面都不加载

**Comment by xiaohucode** [comment]
推特需要登录自己的账号
哔哩哔哩只有搜索
禁漫天堂不加载是触发cf盾了右上角登录,进入登录页面点击完成即可解决（有不会触发cf盾的域名，自己找个替换下）

**Reply by azy00728** [reply]
好的 谢谢作者大大 那个我发的那个添加api的那个网站 你帮我看看呗 看能不能直接套用 那个网站资源不错

**Reply by ixyzi2005** [reply]
大佬怎么写规则，建立个群谈论嘛！还有弄个分组！方便些

**Comment by 13316282905** [comment]
请大家教教我如何使用导入规则
本人一直导入网络上输入网址还是没有添加规则

**Reply by sitontree** [reply]
我的 - 管理站点 - 更多 - 本地导入，从“我的”界面去导入而不是“收藏”界面  我看有另一个叫yssqtao的人回答的，实测管用

**Reply by 13316282905** [reply]
有视频教学吗

**Comment by 13316282905** [comment]
有视频教学吗
